style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -273,6 +273,11 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorBorderDividerDimmed: {
+    description: 'The border color for table row dividers.',
```

Comment:
It looks like this token is not used only for tables. What about just "The border color for table row dividers."?

Reply:
It's also supposed to be used for [the service homepage](https://refresh.cloudscape.aws.dev/examples/react/landing-page.html?awsui-visual-refresh=true).
![Uploading Screenshot 2023-12-07 at 15.29.46.png…]()

Reply:
Now I got what you mean. I'll then change it to `colorBorderRowDivider`. Make sense?